cdap-ui/app/hydrator/leftpanel.less

```diff
@@ -434,11 +442,8 @@ body.theme-cdap.state-hydrator-create {
                 border: 1px solid transparent;
                 color: #666666;
                 line-height: 30px;
-                margin-bottom: 6px;
+                margin-bottom: 15px;
```

Comment:
Existing design:
<img width="712" alt="screen shot 2017-03-29 at 2 27 22 pm" src="https://cloud.githubusercontent.com/assets/1452845/24477439/de366cb4-148b-11e7-80cd-c205454f8b93.png">
<img width="468" alt="screen shot 2017-03-29 at 2 27 14 pm" src="https://cloud.githubusercontent.com/assets/1452845/24477440/de36bcdc-148b-11e7-8631-3b9b1e3d8535.png">

The popover is aligned to the top right of the plugin in the left panel